wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html

```diff
@@ -7,51 +7,66 @@
+            >
+                {% icon name="breadcrumb-expand" class_name="w-w-4 w-h-4" %}
+            </button>
+        {% endif %}
         <div class="w-relative w-h-[50px] w-mr-4 w-top-0 w-z-20 w-flex w-items-center w-flex-row w-flex-1 sm:w-flex-none w-transition w-duration-300">
             <nav class="w-flex w-items-center w-flex-row w-h-full"
                 aria-label="{% trans 'Breadcrumb' %}">
                 <ol class="w-flex w-flex-row w-justify-start w-items-center w-h-full w-pl-0 w-my-0 w-gap-2 sm:w-gap-0 sm:w-space-x-2">
                     {% for page in pages %}
                         {% if page.is_root %}
-                            <li class="{{ breadcrumb_item_classes }} w-max-w-0" data-breadcrumb-item hidden>
-                                <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore_root' %}">
+                            <li
+                                class="{{ breadcrumb_item_classes }} w-max-w-0 {% if is_expanded %}w-max-w-4xl{% endif %}"
+                                data-breadcrumb-item
+                                {% if not is_expanded %}hidden{% endif %}
+                            >
+                                <a
+                                    class="{{ breadcrumb_link_classes }}"
+                                    href="{% if url_root_name %}{% url url_root_name %}{% else %}{% url url_name page.id %}{% endif %}{{ querystring_value }}"
+                                >
                                     {% trans "Root" %}
                                 </a>
                                 {% icon name="arrow-right" class_name=icon_classes %}
                             </li>
                         {% elif forloop.first %}
                             {# For limited-permission users whose breadcrumb starts further down from the root #}
-                            <li class="{{ breadcrumb_item_classes }} w-max-w-0" data-breadcrumb-item hidden>
-                                <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore' page.id %}">
+                            <li
+                                class="{{ breadcrumb_item_classes }} w-max-w-0 {% if is_expanded %}w-max-w-4xl{% endif %}"
+                                data-breadcrumb-item
+                                {% if not is_expanded %}hidden{% endif %}
+                            >
+                                <a class="{{ breadcrumb_link_classes }}" href="{% url url_name page.id %}{{ querystring_value }}">
                                     {% trans "Root" %}
                                 </a>
                                 {% icon name="arrow-right" class_name=icon_classes %}
                             </li>
                         {% elif forloop.last %}
                             <li class="{{ breadcrumb_item_classes }}" {% if trailing_breadcrumb_title %} data-breadcrumb-item hidden {% endif %}>
```

Comment:
looks like this should also have a dynamic `hidden` attribute based on if `is_expanded` is true.

Reply:
however, the issue here is that if `trailing_breadcrumb_title` is used when `is_expanded` is true - it will never be visible as the `hidden` attribute will exist.

The `hidden` then gets removed when the breadcrumbs expand, however when `is_expanded` we just want to show everything.

An example of this behaviour is in the create page breadcrumbs, test this out by temporarily changing the value for that breadcrumb to `is_expanded=True` and check it shows everything correctly.

```suggestion
                            <li class="{{ breadcrumb_item_classes }}" {% if trailing_breadcrumb_title and not is_expanded %} data-breadcrumb-item hidden {% endif %}>
```

**Screenshots**
<img width="624" alt="Screen Shot 2022-06-22 at 8 04 40 am" src="https://user-images.githubusercontent.com/1396140/174904775-47d5ef60-d731-4574-86e0-91ab9a0e4c43.png">

<img width="821" alt="Screen Shot 2022-06-22 at 8 04 44 am" src="https://user-images.githubusercontent.com/1396140/174904764-cc936135-39a9-4525-8a06-54637cd0efaa.png">